examples/swap.ts

```diff
@@ -64,6 +63,8 @@ const swap = async () => {
         chainId,
         paths: sorPaths,
         swapKind,
+        sender,
+        recipient,
```

Comment:
The only downside I see from this refactor is that the consumer will have to provide sender/recipient info even if they want to perform a query.
FE asked me to refactor addLiquidityNested and remove `accountAddress` from the query input, so it's not a requirement.
I know they can come around this by providing ADDRESS_ZERO, but I think it's nice to be explicit on the requirements for each method.
Following that, having sender/recipient on the buildCall only would make more sense.
What do you think?

Reply:
In that case we will should use two different buildCall input types - One for V2 that includes sender/recipient and one for V3 that doesn't. Does that sounds ok? Its a pity we have to force the user to decide but I can't see a way around it.

Reply:
True 😕 
I think that's ok though 👍 
They will need to provide either when building the swap class or when building the call anyway 🤷